Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -241,20 +269,11 @@ void SeedFinder<external_spacepoint_t, platform_t>::filterCandidates(
   float varianceRM = spM.varianceR();
   float varianceZM = spM.varianceZ();
 
-  state.linCircleBottom.clear();
-  state.linCircleTop.clear();
+  auto sorted_bottoms =
+      cotThetaSortIndex(state.compatBottomSP, state.linCircleBottom);
+  auto sorted_tops = cotThetaSortIndex(state.compatTopSP, state.linCircleTop);
```

Comment:
from what I see (am I missing it?) the `cotThetaSortIndex` function makes no use of the first argument? Also, why not simply pass the `state` directly instead of having a list of variables?